src/main/java/org/commcare/formplayer/services/NewFormResponseFactory.java

```diff
@@ -108,7 +108,8 @@ public NewFormResponse getResponse(NewSessionRequestBean bean, String postUrl) t
                 bean.getRestoreAsCaseId(),
                 null,
                 formplayerRemoteInstanceFetcher,
-                bean.getWindowWidth()
+                bean.getWindowWidth(),
+                null
```

Comment:
this doesn't need to be translated ?

Reply:
because `SessionFrame` is null, I believe there is no `commandId`. But I'm not too familiar with this workflow, so happy to follow your lead.